domains/riteshsmooth.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ritesh",
```

this should be your github username
```suggestion
        "username": "ritesh246",
```